__init__.py

```diff
@@ -124,6 +128,12 @@ def initialize(self):
         # Handler For Weather Response
         self.bus.on("skill-ovos-weather.openvoiceos.weather.response", self.update_weather_response)
 
+        # Handler For OCP Player State Tracking
+        self.bus.on("gui.player.media.service.sync.status",
```

Comment:
will need to check, but there may be a more accurate event to use maybe, state events should be emitted live and can be parsed i think. But that can be done later if this is working fine!